lib/ansible/cli/galaxy.py

```diff
@@ -1242,7 +1242,7 @@ def execute_info(self):
     def execute_verify(self, artifacts_manager=None):
 
         collections = context.CLIARGS['args']
-        search_paths = context.CLIARGS['collections_path']
+        search_paths = context.CLIARGS['collections_path'] or AnsibleCollectionConfig.collection_paths
```

Comment:
Before this change, `verify` searched config/sys paths too like `list` (but unlike `list`, cares about order since it stops at the first one). This could just be
```suggestion
        search_paths = AnsibleCollectionConfig.collection_paths
```

Reply:
Resolved